### PR TITLE
BZ 1116864 - Updates to Systems > VHMs section.

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/6387
https://bugzilla.suse.com/show_bug.cgi?id=1116864&GoAheadAndLogIn=1

### CODE REVIEW

**Comment by jcayouette** [comment]
The fix for this was adjusting the section level. The included autoinstallation section was at a level 1 heading, which placed it at the same level as "Systems" By add == to the included autoinstallation file it solved the issue. I have cherrypicked the submitted change to dev from 3.2

**Comment by Loquacity** [comment]
Thanks @jcayouette! I won't close the bug just yet, because there are other issues that still need addressing, but that was the main one :)

**Comment by Loquacity** [comment]
OK, this is complete and ready to merge now.

File: adoc/reference-webui-systems.adoc [hunk 1]
```diff
@@ -2935,22 +2937,31 @@ The following JSON data is an example of the exported information in the file:
...
 /usr/share/doc/packages/virtual-host-gatherer/README.md
 ----
 
+The man page and the `README` file also contain example configuration files for your reference.
+
+ 
+
 === Configuring Virtual Host Managers via XMLRPC API
 
+{productname} ships with a tool called [command]``virtual-host-gatherer`` that can connect to VHMs using their API, and request information about virtual hosts.
```
Comment (moio) [inline adoc/reference-webui-systems.adoc]: `virtual-host-gatherer` has nothing to do with the XMLRPC API, and it feels like the wrong place to describe it here.

The XMLRPC API is (not only in this particular case) meant to give functional equivalents to features present in the Web UI, in a machine/programmer friendly way. So in this case what one can do via Web UI is to view, create, edit and delete VHMs - the XMLRPC API offers the same functionality.

Once the VHMs are stored in the database with their configuration, a nightly/configurable/manually triggerable Taskomatic job will use this information as an input to `virtual-host-gatherer`, which will in turn contact the configured VHMs via their API, collect their data and store it in the database.

From the Taskomatic job's POV it is totally irrelevant whether VHM configuration was created/edited via the Web UI, the XMLRPC API or a combination of both. The same is true for `virtual-host-gatherer`: it is once again totally irrelevant whether its input comes from Taskomatic or a manual invocation (see "File-based").

Conceptually, `virtual-host-gatherer` knows nothing about the XMLRPC API or Taskomatic.

I guess a better solution might be to create a small title for this part.

Hope this helps!

Comment (Loquacity) [reply adoc/reference-webui-systems.adoc]: OK, so I've moved the info around a little and hopefully made it a bit more sane. Let me know what you think.

File: adoc/reference-webui-systems.adoc [hunk 2]
```diff
@@ -2836,12 +2836,20 @@ endif::[]
...
-This tool is automatically invoked via Taskomatic nightly, therefore you need to configure your VHMs via XMLRPC APIs. [command]``virtual-host-gatherer`` maintains the concept of optional modules, where each module enables a specific Virtual Host Manager.
+Third party hypervisors and hypervisor managers such as VMWare vCenter are called "`Virtual Host Managers`" (VHM).
+VHMs can manage one or more virtual hosts, and each virtual host can contain one or more virtual guests.
+
+To create a VHM, navigate to menu:Systems[Virtual Host Managers] in the {webui}.
+In the upper right of the page, click btn:[Create] and select either [guimenu]``VMware-based`` or [guimenu]``File-based``.
+Continue according to one of the following procedures.
+
+After it has been created and configured, you can begin data collection from your VHMs manually through the {webui}, by navigating to menu:Systems[Virtual Host Managers], selecting the appropriate VHM, and clicking btn:[Refresh Data].
```
Comment (moio) [inline adoc/reference-webui-systems.adoc]: Just to be extra clear - clicking on "Refresh Data" has the same effect of waiting a nightly Taskomatic round - it results in `virtual-host-gatherer` being run.

File: adoc/reference-webui-systems.adoc [hunk 3]
```diff
@@ -2867,7 +2875,7 @@ image::systems_virtual_host_managers_vmware.png[scaledwidth=80%]
 [[ref.webui.systems.virt-host-managers.file]]
 === File-Based
 
-In a VMWare environment where direct connection to the SUSE Manager Server is not possible, a JSON file can be exported from the ESXi or vSphere host and later imported into {productname} via this option.
+In a VMWare environment where direct connection to the {susemgr}} is not possible, a JSON file can be exported from the ESXi or vSphere host and later imported into {productname}.
```
Comment (moio) [inline adoc/reference-webui-systems.adoc]:
```suggestion
In a VMWare environment where direct connection to the {susemgr} is not possible, a JSON file can be exported from the ESXi or vSphere host and later imported into {productname}.
```

File: adoc/reference-webui-systems.adoc [hunk 4]
```diff
@@ -2935,22 +2943,26 @@ The following JSON data is an example of the exported information in the file:
...
 ----
 
-=== Configuring Virtual Host Managers via XMLRPC API
+The man page and the `README` file also contain example configuration files for your reference.
+
+
+
+=== Configuring Virtual Host Managers using the XMLRPC API
 
+In order to manage your VHMs with XMLRPC, you will need to configure them to use the XMLRPC APIs.
```
Comment (moio) [inline adoc/reference-webui-systems.adoc]: VHMs do not really use the XMLRPC APIs, rather, VHMs can be configured either manually via the Web UI or programmatically via the XMLRPC APIs.

**Pull request merged**